### PR TITLE
updated to JUPnP 2.1.0.RC1, which requires new config settings

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
fixes https://github.com/openhab/openhab-distro/issues/68

File: launch/home/etc/services.cfg [hunk 1]
```diff
@@ -20,8 +20,8 @@ org.openhab.autoapprove:enabled=false
 org.eclipse.smarthome.binding.sonos:opmlPartnerID=IAeIhU42
 
 org.jupnp:multicastResponsePort=0
-org.jupnp:streamListenPort=8080
-org.jupnp:threadPoolSize=200
+org.jupnp:asyncThreadPoolSize=30
+org.jupnp:threadPoolSize=15
```
Comment (dvanherbergen) [inline launch/home/etc/services.cfg]: I'm not an expert on how upnp works, but 45 threads in total, isn't that a bit much?

Comment (kaikreuzer) [inline launch/home/etc/services.cfg]: No, it is a pretty good setting.
Note that up to now, we had 200 threads that were always started at startup. Now we have pools only temporarily increase if necessary, up to a maximum of 45 threads. (and when running a upnp discovery, you can easily have 500-1000 incoming requests to process)

Comment (dvanherbergen) [inline launch/home/etc/services.cfg]: Thanks for the explanation.

**Pull request merged**